README.md

```diff
@@ -10,15 +10,15 @@ Steps to run a local copy of SpartaHackAPI
     ```
       pip3 install virtualenvwrapper
     ```  
-   And add the following three lines to your shell startup file (.bashrc, .profile, etc.)  
+   And add the following three lines to your shell startup file (.zshrc, .bashrc, .profile, etc.)  
     ```
-      export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+      export VIRTUALENVWRAPPER_PYTHON=/usr/local/bin/python3
       export WORKON_HOME=~/.virtualenvs
       source /usr/local/bin/virtualenvwrapper.sh
     ```  
 2. Clone the API and cd into it  
-3. Create a virtualenv using `mkvirtualenv` using the virtualenvwrapper  
-4. Switch the python interpreter to the virtualenv using `workon virtualenv_name`  
+3. Create a virtualenv using `mkvirtualenv [DEST]` using the virtualenvwrapper  
+4. Switch the python interpreter to the virtualenv using `workon [VIRTUALENV_NAME]`  
```

Comment:
looks good!

Reply:
also, the Dest part, should be the name right, you should go and change that